custom_model_runner/datarobot_drum/drum/language_predictors/java_predictor/java_predictor.py

```diff
@@ -176,8 +177,10 @@ def _stop_py4j(self):
         self._cleanup()
 
     def _run_java_server_entry_point(self):
-
         java_cp = ":".join(self._jar_files)
+        custom_class_path = os.environ.get(EnvVarNames.DRUM_JAVA_CUSTOM_CLASS_PATH)
+        if custom_class_path is not None:
+            java_cp = "{}:{}".format(java_cp, custom_class_path)
```

Comment:
A log message here would be good to. To show the content of the custom class path.

Reply:
Done